### DISCUSSION TITLE
Is there any way to disable the mode and what I assume is configuration displays on the bottom bar?

### DISCUSSION THREAD

**Discussion by flashgnash** [zellij-org/zellij#3586] [Q&A]
Have accepted not being able to remove the brand from the bottom left, but is it possible to remove the mode (NORMAL) and layout (BASE) displays from the tab bar?

**Comment by flashgnash** [comment]
Also, is it possible to have the whole tab bar hidden when there's only one tab open?

**Reply by Lockszmith-GH** [reply]
I'm not sure about this one, but the bottom bar is a plugin, check the asweome zelilj list for alternative plugins, if I recall correctly there is one status-bar plugin that allows customization

**Reply by flashgnash** [reply]
I have seen this, I was hoping not to have to use custom plugins to keep the complexity down (also unsure whether they're packaged for nix yet)
